src/carbon-calculation/interfaces/OCLCAJobs.ts

```diff
@@ -0,0 +1,153 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { CarbonUploadState } from "../../common/CarbonCalculation";
+import { Link } from "../../common/Links";
+
+export interface OCLCAJobCreate {
+  /**
+   * Report identifier used to upload report data to One Click LCA.
+   * @type {string}
+   * @memberof OCLCAJobCreate
+   */
+  reportId: string;
+  /**
+   * One Click LCA token. Acquiring a token is possible using One Click LCA APIs. Contact api@oneclicklca.com for support.
+   * @type {string}
+   * @memberof OCLCAJobCreate
+   */
+  token: string;
+}
+/**
```

Comment:
Add empty line between previous interface closing brace and next interface documentation comment.

Reply:
Done.